### PR TITLE
fix: don't forward to next after responding

### PR DESCRIPTION
**Pull request by ForsakenHarmony**

### CODE REVIEW

**Review by marvinhagemeister — APPROVED** [review]

File: packages/wmr/src/start.js [hunk 1]
```diff
@@ -166,7 +166,6 @@ const injectWmrMiddleware = ({ cwd }) => {
 		} catch (e) {
 			next();
 		}
-		next();
```
Comment (marvinhagemeister) [inline packages/wmr/src/start.js:169]: Wait on second thought: Do we still need to call `next()` in the success case when we don't end the request?

Comment (ForsakenHarmony) [reply packages/wmr/src/start.js:169]: Oh right, we do, kinda missed that if 🙈

Comment (developit) [reply packages/wmr/src/start.js:169]: I think this needs to be:

```js
try {
  if (...) {
    res.end(result);
    return;
  }
} catch (e) {}
next();
```

Comment (ForsakenHarmony) [reply packages/wmr/src/start.js:169]: Oh, I moved out the if now

**Review by marvinhagemeister — APPROVED** [review]

**Pull request merged**